interface/rest/error_handler.go

```diff
@@ -0,0 +1,20 @@
+package rest
+
+import (
+	"net/http"
+
+	rest "github.com/der-nackte-halloumi/api/interface/rest/models"
+)
+
+type errorHandler struct{}
```

Comment:
Would you want to put a logger or something here? Maybe you're handling that at a higher level-- just checking.

Reply:
Don’t have a logger yet, but thanks for the hint! That basically means, I would inject a logger here to log the errors in one central place?

Reply:
Yeah, just so you can track/find 404s.